controlplane/kubeadm/internal/workload_cluster_rbac.go

```diff
@@ -35,6 +36,14 @@ const (
 	// GetNodesClusterRoleName defines the name of the ClusterRole and ClusterRoleBinding to get nodes.
 	GetNodesClusterRoleName = "kubeadm:get-nodes"
 
+	// SuperAdminKubeConfigFileName defines name for the kubeconfig aimed to be used by the super-admin of the cluster.
+	SuperAdminKubeConfigFileName = "super-admin.conf"
```

Comment:
We should be able to import these constants once 1.29 is released, but for now we copy them like these other constants. Not sure if it's worth the import.

Reply:
Not sure if we can import from this part of k/k without getting a dependency on k/k itself

Reply:
This specific constant seems to be unused, let's delete it if we don't use it